### PR TITLE
Clean up fvm mocks

### PR DESCRIPTION
**Pull request by pattyshack**
1. Removed dead mock directory fvm/mock
2. Updated makefile to wipe environment/mock directory before regenerating mocks
3. Updated test to stop using dead mocks

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -162,9 +162,8 @@ generate-mocks: install-mock-generators
 	mockery --name '.*' --dir=engine/execution/state --case=underscore --output="./engine/execution/state/mock" --outpkg="mock"
 	mockery --name '.*' --dir=engine/consensus --case=underscore --output="./engine/consensus/mock" --outpkg="mock"
 	mockery --name '.*' --dir=engine/consensus/approvals --case=underscore --output="./engine/consensus/approvals/mock" --outpkg="mock"
-	mockery --name '.*' --dir=fvm --case=underscore --output="./fvm/mock" --outpkg="mock"
+	rm -rf ./fvm/environment/mock
```
Comment (Tonix517) [inline Makefile:165]: why do we need this rm only for this directory? doesn't mockery simply overwrite existing files?

Comment (pattyshack) [reply Makefile:165]: it doesn't clean up after file renames

Comment (pattyshack) [reply Makefile:165]: we should probably wipe every mock directories, but that require moving files first (some mock directories contain non-generated code)

**Review by janezpodhostnik — APPROVED** [review]

**Review by Tonix517 — APPROVED** [review]

**Comment by pattyshack** [comment]
bors merge

**Comment by pattyshack** [comment]
bors merge

**Comment by pattyshack** [comment]
bors merge

**Comment by janezpodhostnik** [comment]
bors retry

**Comment by janezpodhostnik** [comment]
bors merge

**Comment by janezpodhostnik** [comment]
bors merge

**Comment by janezpodhostnik** [comment]
finally!

**Comment by pattyshack** [comment]
thx!

**Pull request merged**